### PR TITLE
Use templates for python version dependencies

### PR DESCRIPTION
**Pull request by kpayson64**

### CODE REVIEW

**Comment by nicolasnoble** [comment]
Ah. That's kind of urgent :)

File: src/python/grpcio_health_checking/setup.py [hunk 1]
```diff
@@ -46,12 +46,12 @@
...
 
 SETUP_REQUIRES = (
-    'grpcio-tools>=0.15.0',
+    'grpcio-tools>={version}'.format(version=grpc_version.VERSION),
 )
 
 INSTALL_REQUIRES = (
     'protobuf>=3.0.0',
-    'grpcio>=0.15.0',
+    'grpcio>={version}'.format(version=grpc_version.VERSION)',
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio_health_checking/setup.py]: Nonsense single-quote near the end of this line, right? Causes a parse error?

Comment (kpayson64) [inline src/python/grpcio_health_checking/setup.py]: Good catch, thanks

Comment (nicolasnoble) [inline src/python/grpcio_health_checking/setup.py]: Are we fixing that one ?

Comment (kpayson64) [inline src/python/grpcio_health_checking/setup.py]: Yes, I'm running the tests locally this time and that typically takes 10+ minutes.

**Comment by kpayson64** [comment]
Ok tests pass locally now.

**Comment by nathanielmanistaatgoogle** [comment]
Commit message should describe and justify the `SETUP_REQUIRES` change.

**Comment by kpayson64** [comment]
Updated commit message.

**Comment by nathanielmanistaatgoogle** [comment]
Luhguhtum and thank you.

**Pull request merged**